src/components/input/DurationInput.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Box, NumericInput } from '../../';
+
+const DurationInput = ({ value, onChange, onKeyDown }) => {
```

Comment:
Is this the best fitting name for this component? 

For the current use case, bulk time tracking modal, this will be used to set the duration. But in the future, this might be used to set the start and end time of something. In that case, it's not a duration anymore.
On the other hand, keeping this name might prevent misusing this component for things it isn't meant for.

Reply:
Good question, for the start and end time the `TimeInput` component should be used, this would then be limited to 24h, where as this one isn't

Reply:
makes sense! :+1: